apps/ledger-live-desktop/src/renderer/screens/settings/sections/General/walletSync/WalletSyncRow.tsx

```diff
@@ -1,25 +1,19 @@
-import { useFeature } from "@ledgerhq/live-common/featureFlags/index";
 import React, { useState } from "react";
+import { useSelector } from "react-redux";
 import Button from "~/renderer/components/Button";
 import { SideDrawer } from "~/renderer/components/SideDrawer";
-
-const SideContentActivateWalletSync = () => {
-  return <div>Activate Wallet Sync</div>;
-};
-
-const SideContentWalletSync = () => {
-  return <div>Wallet Sync</div>;
-};
+import { walletSyncSelector } from "~/renderer/reducers/walletSync";
+import WalletSyncActivation from "./sideContent/Activation";
+import WalletSyncManage from "./sideContent/Manage";
 
 const WalletSyncRow = () => {
-  const lldWalletSync = useFeature("lldWalletSync");
-
+  const walletSync = useSelector(walletSyncSelector);
```

Comment:
you can create a selector for `activated` only, in order to not "export" all the state

Reply:
Felt like it wasn't necessary right now.